composer.json

```diff
@@ -23,16 +23,16 @@
 
   "require": {
     "php": ">=7.3",
-    "psr/log": "~1.0",
+    "psr/log": "1 - 3",
     "predis/predis": "^2.0",
-    "symfony/yaml": ">=5.3"
+    "symfony/yaml": "5.3 - 6"
```

Comment:
With this range installation SF of 6.1 is not possible.
https://getcomposer.org/doc/articles/versions.md#hyphenated-version-range-